Rakefile.rb

```diff
@@ -48,6 +48,36 @@ def name_from(json)
   orgs.first[:name]
 end
 
+def file_to_commit_metadata
+  # There's about 2MB of data returned by this command, so rather than
+  # using backticks to turning it into a string and then splitting it,
+  # parse the output line-by-line. As a bonus this doesn't
+  # unnecessarily invoke a shell.
+  command = ['git', '--no-pager', 'log', '--name-only',  '--format=%h|%at', '--', 'data']
+  result = {}
+  sha, timestamp = nil
```

Comment:
This does _work_, but it feels odd to me because it looks like you're expecting the assignment to apply to both elements on the left hand side, with the implication that after `foo, bar = 1`, both `foo` and `bar` would each be set to `1`. 

`sha, timestamp = nil, nil` would be more correct, I think, but as you never actually do anything with those variable separately, I suspect a cleaner approach would be to keep them paired as a `commit_details` hash instead:

```
      f.each_line do |line|
        commit_match = line.match /^([a-f\d]+)\|(\d+)$/
        if commit_match
          commit_details = { sha: commit_match[1], timestamp: commit_match[2] }
        else
          result[line.strip] ||= commit_details
        end
      end
```

Though that might be clearer still with named captures:
```
      f.each_line do |line|
        commit_match = line.match /^(?<sha>[a-f\d]+)\|(?<timestamp>\d+)$/
        if commit_match
          commit_details = { sha: commit_match[:sha], timestamp: commit_match[:timestamp] }
        else
          result[line.strip] ||= commit_details
        end
      end
```

(NB: For maximum readability I'd be tempted to actually monkey patch MatchData with: 
```ruby
def to_hash
  Hash[ (self.names.map { |n| n.to_sym }).zip( self.captures ) ]
end
```

But that might be a step too far…)